src/util/dsp/LogSlider.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+namespace otto::util::dsp {
+  class LogSlider
+  {
+  public:
+      LogSlider();
+      ~LogSlider();
+
+      double proportionOfLengthToValue(double proportion);
```

Comment:
Id like to see some documentation for these functions. like `\param proportion must be in range [0;1]`

Also, we use 32 bit floats for most things, not sure if double is ever necessary